test/clojure-mode-util-test.el

```diff
@@ -82,6 +82,21 @@
       (expect (clojure-find-ns) :to-equal "foo"))
     (with-clojure-buffer "(ns ^:bar ^:baz foo)"
       (expect (clojure-find-ns) :to-equal "foo")))
+  (it "should find namespaces with spaces before ns form"
```

Comment:
I'd suggest to also include the following case:

```clj
(comment
  (ns foo))
```

...Asserting that `(clojure-find-ns)` is nil.

That would help us prevent anything like an infinite loop.

Reply:
Made both of the suggested changes.
Please squash the commits while merging.